sources/packages/backend/libs/auth/src/services/keycloak/keycloak.service.ts

```diff
@@ -41,9 +39,11 @@ export class KeycloakService {
    * injection framework.
    */
   public static get shared(): KeycloakService {
+    const logger = new LoggerService();
+    logger.setContext(KeycloakService.name);
```

Comment:
As we can pass the parent class to the constructor, what about declaring this way? `const logger = new LoggerService(KeycloakService);`

Reply:
It would not result in the same output, the constructor is expecting an instance.
I was going to create an overload to accept the string, but there are only two places not using dependency injection right now, so I did not invest time in it.